src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -105,12 +106,50 @@ export const IframeRuntime: React.FC<IProps> =
       addListener("showModal", () => {
         toggleModal();
       });
-      post("initInteractive", {
-        mode: report ? "report" : "runtime",
+      // note: many of the values here are placeholders that require further
+      // consideration to determine whether there are more appropriate values.
+      const baseProps: Omit<IReportInitInteractive, "mode"> = {
```

Comment:
You've used `IreportInitInteractive` as that's the smallest interface, right?

Reply:
Yes, of the two interfaces of interest here it has all of the shared properties and only one report-specific property.